docs/en/stack/security/get-started-security.asciidoc

```diff
@@ -0,0 +1,311 @@
+clusters). 
+
+To help you get up and running, there are built-in roles. Go to the 
+*Management / Security / Roles* page to see them:
+
+[role="screenshot"]
+image::security/images/management-roles.jpg["Role management screenshot in Kibana"]
+
+Select a role to see more information about what privileges it provides. For 
+example, if you select the `kibana_user` role, you will see that it grants 
+`manage`, `read`, `index`, and `delete` privileges on the `.kibana*` indices. 
+
+If you want to learn more about authorization and roles, see <<authorization>>.
+
+[float]
+[[gs-roles]]
+=== Create roles in {kib}
+
+If you completed all of the steps in <<get-started-elastic-stack>>, you should 
+have {metricbeat} data stored in {es}. Let's create two roles that grant 
+different levels of access to that data. 
+
+Go to the *Management / Security / Roles* page and click *Create role*. 
+
+Create a `metricbeat_reader` role that has `read` and `view_index_metadata` 
+privileges on the `metricbeat-*` indices:
+
+[role="screenshot"]
+image::security/images/create-reader-role.jpg["Creating a role in Kibana"]
+
+Create a `metricbeat_writer` role that has `manage_index_templates` and `monitor` 
+cluster privileges, as well as `write`, `delete`, and `create_index` privileges
+on the `metricbeat-*` indices:
+
+// TBD: The Beats docs don't mention the need for the delete privilege, whereas 
+// the Logstash docs do. Which should be used in this case?
+
+[role="screenshot"]
+image::security/images/create-writer-role.jpg["Creating another role in Kibana"]
+
+Now go back to the *Management / Security / Users* page and assign these roles 
+to the appropriate users. Assign the `metricbeat_reader` role to your personal 
+user.  Assign the `metricbeat_writer` role to the `metricbeat_internal` user.
+
+The list of users should now contain all of the built-in users as well as the 
+two you created. It should also show the appropriate roles for your users:
+
+[role="screenshot"]
+image::security/images/management-users.jpg["User management screenshot in Kibana"]
+
+[float]
+[[gs-logstash-security]]
+=== Add users in {ls}
+
+In order for {ls} to send data successfully to {es}, you must configure its 
+authentication credentials. You must configure credentials separately for each 
+of the {es} plugins in your {ls} configuration file.  For example, update the `demo-metrics-pipeline.conf` file in your {ls} directory:
+
+[source,ruby]
+----
+input {
+  beats {
+    port => 5044
+  }
+}
+
+filter {
+  if [system][process] {
+    if [system][process][cmdline] {
+      grok {
+        match => {
+          "[system][process][cmdline]" => "^%{PATH:[system][process][cmdline_path]}"
+        }
+        remove_field => "[system][process][cmdline]"
+      }
+    }
+  }
+}
+
+output {
+  elasticsearch {
+    hosts => "localhost:9200"
+    manage_template => false
+    index => "%{[@metadata][beat]}-%{[@metadata][version]}-%{+YYYY.MM.dd}"
+    user => metricbeat_internal <1>
```

Comment:
I'd put double quotes around the username and password: `user => "metricbeat_internal"` and `password => your_password`

The last time I checked, if the password contains dashes, it won't work without the double quotes (see https://github.com/elastic/logstash/issues/9618). I think this is true for user, too, but I haven't tested it.

Reply:
Also (this comment applies to other sections, too), I wonder if this tutorial should be showing how to use the secrets keystores (in ES and LS), too, so that users don't have to hard code values in their configs. I know this makes the tutorial more complicated, but most users do not want to have passwords revealed in config files, so it's a very common use case.